charts/fluent-bit/templates/daemonset.yaml

```diff
@@ -27,7 +27,11 @@ spec:
   template:
     metadata:
       labels:
-        {{- include "fluent-bit.selectorLabels" . | nindent 8 }}
+        {{- if .Values.verbosePodLabels.enabled }}
+          {{- include "fluent-bit.verbosePodLabels" . | nindent 8 }}
+        {{- else }}
+          {{- include "fluent-bit.selectorLabels" . | nindent 8 }}
+        {{- end }}
```

Comment:
```suggestion
        {{- include (ternary "fluent-bit.verbosePodLabels" "fluent-bit.selectorLabels" .Values.verbosePodLabels)  . | nindent 8 }}
```
I think this would look a bit cleaner.